tests/messages/test_pofile.py

```diff
@@ -1014,3 +1014,10 @@ def test_issue_1087():
 "Language: \n"
 ''')
     assert pofile.read_po(buf).locale is None
+
+def test_issue_1134():
+    buf = StringIO('''msgid "this is an invalid po file"
+msgstr "hello world"
+msgid "msgid without str"''')
+    with pytest.raises(pofile.PoFileError):
```

Comment:
`PoFileError` is raised for various reasons, I'd also assert that the exception message matches ours:

```python
with pytest.raises(pofile.PoFileError) as excinfo:
    pofile.read_po(buf, abort_invalid=True)
assert str(excinfo.value) == "..." 
```